README.md

```diff
@@ -9,16 +9,20 @@ Where possible, the services available on the client groups the API into logical
 ## Usage
 
 ``` go
-import "github.com/rpip/paystack-go"
+import (
+    "github.com/rpip/paystack-go"
```

Comment:
try importing the package this way, it's less ambiguous. 

import (
   paystack "github.com/rpip/paystack-go"
   "strconv"
    "fmt"
)